src/app/helpers/pgp.js

```diff
@@ -0,0 +1,73 @@
+import { RECIPIENT_TYPE, KEY_FLAGS } from 'proton-shared/lib/constants';
+import { arrayToBinaryString, encodeBase64, getKeys, isExpiredKey, stripArmor } from 'pmcrypto';
+
+const { TYPE_INTERNAL } = RECIPIENT_TYPE;
+const { ENABLE_ENCRYPTION } = KEY_FLAGS;
+
+/**
+ * Check if it's an internal contact
+ * @param {Integer} config.RecipientType from API
+ * @returns {Boolean}
+ */
+export const isInternalUser = ({ RecipientType }) => RecipientType === TYPE_INTERNAL;
+
+/**
+ * Test if every keys are not enabled
+ * @param {Object} config from API
+ * @returns {Boolean}
+ */
+export const isDisabledUser = (config) =>
+    isInternalUser(config) && config.Keys.every(({ Flags }) => !(Flags & ENABLE_ENCRYPTION));
```

Comment:
instead of every you can use some with the opposite condition, as it will fail faster ;) if every returns false here some will always be more efficient ;)